src/goMain.ts

```diff
@@ -367,8 +367,29 @@ export function activate(ctx: vscode.ExtensionContext): void {
 	}));
 
 	ctx.subscriptions.push(vscode.commands.registerCommand('go.show.commands', () => {
-		vscode.window.showQuickPick(getExtensionCommands().map(x => x.title)).then(cmd => {
-			let selectedCmd = getExtensionCommands().find(x => x.title === cmd);
+		let extCommands = getExtensionCommands()
+		extCommands.push({
+			command : 'editor.action.goToTypeDefinition',
+			title : 'Go : Go to Type Definition',
+			description : 'Go : invokes the GoDefinitionProvider'
+		})
+		extCommands.push({
+			command : 'editor.action.goToImplementation',
+			title : 'Go : Go to Implementation',
+			description : 'Go : invokes the GoImplementationProvider'
+		})
```

Comment:
Missing semicolon for each of the `push` is causing linting errors in the build

Reply:
thanks. i am new to type script.
is there a way to run such checks locally? before sending the PR

Reply:
Yes, you can run `npm run lint` in the terminal